<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Input Homework Scores&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homework 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homework 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homework 3</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HW4">
     <item>
      <widget class="QSlider" name="hw4slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw4box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homework 4</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HW5">
     <item>
      <widget class="QSlider" name="hw5slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw5box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homework 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homework 6</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HW6">
     <item>
      <widget class="QSlider" name="hw6slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw6box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Homework 7</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HW7">
     <item>
      <widget class="QSlider" name="hw7slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw7box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>20</width>
      <height>421</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Input Midterm Scores&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>50</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="MT1">
     <item>
      <widget class="QSlider" name="mt1slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mt1box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>130</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="MT2">
     <item>
      <widget class="QSlider" name="mt2slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mt2box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Input Final Exam Score&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="FINAL">
     <item>
      <widget class="QSlider" name="feslider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fnlbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>340</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Choose grading method&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>20</width>
      <height>81</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>530</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600; color:#ff0000;&quot;&gt;YOUR GRADE IS...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calculate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>CALCULATE</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>133</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="GradeMethod">
     <item>
      <widget class="QRadioButton" name="A">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Method A</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="B">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Method B</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HW1">
     <item>
      <widget class="QSlider" name="hw1slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw1box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HW2">
     <item>
      <widget class="QSlider" name="hw2slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw2box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HW3">
     <item>
      <widget class="QSlider" name="hw3slider">
       <property name="cursor">
        <cursorShape>SizeHorCursor</cursorShape>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw3box">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="GradeTotal">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>OCR A Extended</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>15</number>
    </property>
    <property name="text">
     <string>........</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>hw1slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw1box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw1box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw1slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw3slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw3box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw3box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw3slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw4slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw4box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw4box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw4slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw5slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw5box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw5box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw5slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw6slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw6box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>385</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw6box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw6slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw7box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw7slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mt1slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mt1box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mt1box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mt1slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>453</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mt2slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mt2box</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>464</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mt2box</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mt2slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>feslider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fnlbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>461</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fnlbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>feslider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
